Gravity/Gravity.Test/TestClasses/GravityLevelOne.cs

```diff
@@ -42,7 +42,7 @@ public class GravityLevelOne : BaseDto
 		[RelativityObjectField("C3336C2C-5A97-4EB1-A3A4-929D79658B8D", RdoFieldType.SingleObject, typeof(GravityLevel2))]
 		public int GravityLevel2ArtifactId { get; set; }
 
-		[RelativitySingleObject("C3336C2C-5A97-4EB1-A3A4-929D79658B8D")]
+		[RelativitySingleObject("C3336C2C-5A97-4EB1-A3A4-929D79658B8D", typeof(GravityLevel2))]
```

Comment:
I thought we removed this because the type is in the declaration of the object itself. as well as got rid of  [RelativityObjectField("C3336C2C-5A97-4EB1-A3A4-929D79658B8D", RdoFieldType.SingleObject, typeof(GravityLevel2))]
 		public int GravityLevel2ArtifactId { get; set; }

Please take a look at d612d5db2bf4b36f50ce7587c87a5aa53d452eac